.gitignore

```diff
@@ -0,0 +1,38 @@
+build/*.*
+build/CMakeFiles/2.8.12.2/CMakeASMCompiler.cmake
+build/CMakeFiles/2.8.12.2/CMakeCCompiler.cmake
+build/CMakeFiles/2.8.12.2/CMakeCXXCompiler.cmake
+build/CMakeFiles/2.8.12.2/CMakeDetermineCompilerABI_C.bin
+build/CMakeFiles/2.8.12.2/CMakeDetermineCompilerABI_CXX.bin
+build/CMakeFiles/2.8.12.2/CMakeRCCompiler.cmake
+build/CMakeFiles/2.8.12.2/CMakeSystem.cmake
+build/CMakeFiles/2.8.12.2/CompilerIdC/CMakeCCompilerId.c
+build/CMakeFiles/2.8.12.2/CompilerIdC/CMakeCCompilerId.exe
+build/CMakeFiles/2.8.12.2/CompilerIdC/CMakeCCompilerId.obj
+build/CMakeFiles/2.8.12.2/CompilerIdCXX/CMakeCXXCompilerId.cpp
+build/CMakeFiles/2.8.12.2/CompilerIdCXX/CMakeCXXCompilerId.exe
+build/CMakeFiles/2.8.12.2/CompilerIdCXX/CMakeCXXCompilerId.obj
+build/CMakeFiles/CheckTypeSize/SIZEOF_VOID_P.bin
+build/CMakeFiles/CheckTypeSize/SIZEOF_VOID_P.c
+build/CMakeFiles/cmake.check_cache
+build/CMakeFiles/CMakeDirectoryInformation.cmake
+build/CMakeFiles/CMakeError.log
+build/CMakeFiles/CMakeOutput.log
+build/CMakeFiles/CMakeRuleHashes.txt
+build/CMakeFiles/Makefile.cmake
+build/CMakeFiles/Makefile2
+build/CMakeFiles/progress.marks
+build/CMakeFiles/TargetDirectories.txt
+build/luarocks/cmake_install.cmake
+build/luarocks/CMakeFiles/CMakeDirectoryInformation.cmake
+build/luarocks/CMakeFiles/progress.marks
+build/luarocks/Makefile
+build/luarocks/src/bin/luarocks.bat
+build/luarocks/src/bin/luarocks-admin.bat
+build/luarocks/src/luarocks/config.lua
+build/luarocks/src/luarocks/site_config.lua
+build/Makefile
+build/LUA_DEV
+bld07/*
```

Comment:
bld07/ is non-standard :-)  fine with it personally, but I reckon it'd be cleaner to remove it, on the hwole